src/server/protocol.ts

```diff
@@ -3413,6 +3432,10 @@ namespace ts.server.protocol {
          * in addition to `const objectLiteral: T = { foo }`.
          */
         readonly includeCompletionsWithObjectLiteralMethodSnippets?: boolean;
+        /**
+         * Indicates whether {@link CompletionEntry.labelDetails completion entry label details} are supported.

```

Comment:
Is that actually how `link` works? Bizarre.

Anyhow



```suggestion
         * Indicates whether {@link CompletionEntry.labelDetails completion entry label details} are supported.
         * If not, contents of `labelDetails` may be included in the {@link CompletionEntry.name} property.
```